packages/web-components/package.json

```diff
@@ -19,9 +19,9 @@
     "build": "stencil build",
     "start": "stencil build --dev --watch --serve",
     "build:watch": "npm run build -- --watch",
-    "test": "stencil test --spec --passWithNoTests",
+    "test": "stencil test --spec --passWithNoTests --silent",
```

Comment:
It seems to prevent any messages in the console, so not ideal. The remainder of the console warnings are things that I couldn't find fixes for to get rid of them entirely. I'll remove the flag so long as we're happy that there will still be some console warnings